test/infra/infra-dbdeployer-mysql84-gr/docker-compose-destroy.bash

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+set -o pipefail
+pushd $(dirname $0) &>/dev/null
+trap 'popd &>/dev/null' EXIT
+set -a; . .env; set +a
```

Shellcheck (0.11.0)

[info] Not following: .env was not specified as input (see shellcheck -x).

(SC1091)